lib/FinderNavigation/FinderItem.js

```diff
@@ -8,13 +8,20 @@ const FinderItem = ({
   hoverSettings,
   selected,
   children,
+  noIndentation,
   ...rest
 }) => {
-  const classNames = cx(`finder__item ${className}`, {
-    'is-active': active,
-    'finder__item--hover-settings': hoverSettings,
-    'finder__item--selected': selected
-  });
+  const classNames = cx(
+    `${
+      noIndentation ? 'finder__item-no-indentation' : 'finder__item'
```

Comment:
Stylistically I think I'd prefer this ternary split out to a const above, just to keep the classnames set up cleaner.